RoslynPluginGenerator/AnalyzerPluginGenerator.cs

```diff
@@ -139,6 +149,25 @@ private bool TryGenerateRulesFile(string packageDirectory, string nuGetDirectory
             return success;
         }
 
+        private IEnumerable<string> GetAssembliesFromPackage(IPackage package, string baseTempDir)
+        {
+            // We can't directly get the paths to the files in package so
+            // we have to extract them first
+            string extractDir = CreateSubDirectory(baseTempDir, ".extract");
+            this.logger.LogDebug(UIResources.APG_ExtractingPackageFiles, extractDir);
+            PhysicalFileSystem fileSystem = new PhysicalFileSystem(extractDir);
+            package.ExtractContents(fileSystem, ".");
+
+            string[] files = Directory.GetFiles(extractDir, "*.*", SearchOption.AllDirectories);
+            return files.Where(f => IsAssembly(f));
+        }
+
+        private static bool IsAssembly(string filePath)
+        {
+            return filePath.EndsWith(".dll", StringComparison.OrdinalIgnoreCase) ||
+                filePath.EndsWith(".exe", StringComparison.OrdinalIgnoreCase);
```

Comment:
I don't think we want to support .winmd files; they target the Windows RT and it doesn't seem likely that people will be writing Roslyn analyzers using WIndows RT. Mind you, it's not likely that many analyzers will be in packaged in exes either.